flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/agg/batch/AggCodeGenHelper.scala

```diff
@@ -93,6 +93,8 @@ object AggCodeGenHelper {
       .asInstanceOf[Map[AggregateFunction[_, _], String]]
   }
 
+  /** @deprecated Use [[RowTypeUtils#projectRowType]] instead. */
+  @deprecated
   def projectRowType(rowType: RowType, mapping: Array[Int]): RowType = {
```

Comment:
remove this method since it's only be used in tests

Reply:
Ok, I'll remove it directly.